### PR TITLE
Add DAI

### LINKED ISSUE #1498: Add DAI to our bridge
As a User I want to be able to bridge DAI

- [x] Add DAI to the token list 
- [x] Create DAI token contract config file 
- [x] add DAI token to the chainconfig file
- [x] add DAI to runyourown agent docus

### PR DESCRIPTION
**Pull request by manuelwedler**
closes #1498

adds DAI token to all necessary configurations. Additionally updates the mainnet deployment configuration for the Arbitrum deployment which was done without updating our repo.

Caution! Merging this in its current state will make the frontend build with DAI. So we couldn't make another frontend release before releasing DAI functionality. So we should make a decision first when we want to release DAI.

### CODE REVIEW

**Review by GabrielBuragev — CHANGES_REQUESTED** [review]
Before merging this we should "whitelist" the DAI token address inside our contracts by calling the `RequestManager.updateToken` & `FillManager.updateToken` (on each rollup) with the respective values for the config parameters.

@fredo please take care of providing these values or even configuring the contracts.

Also, when merging this one, make sure we have this PR merged too: https://github.com/beamer-bridge/run-your-own-agent/pull/16

Rough estimation on the deployment of this from @andfletcher: 
> Lets do it end of sprint so next week thursday

**Comment by fredo** [comment]
> Before merging this we should "whitelist" the DAI token address inside our contracts by calling the `RequestManager.updateToken` & `FillManager.updateToken` (on each rollup) with the respective values for the config parameters.
> 
> @fredo please take care of providing these values or even configuring the contracts.
> 
> Also, when merging this one, make sure we have this PR merged too: [beamer-bridge/run-your-own-agent#16](https://github.com/beamer-bridge/run-your-own-agent/pull/16)
> 
> Rough estimation on the deployment of this from @andfletcher:
> 
´

We only need to update the RequestManager. FillManager.updateToken does not exist. 
Unless it is decided otherwise, values will be

```
transferLimit: 2000_000_000_000_000_000_000
minLpFee: 1_000_000_000_000_000_000
lpFee: 3000
protocolFee: 0
```

File: frontend/config/tokens/DAI.json [hunk 1]
```diff
@@ -0,0 +1,10 @@
+{
+  "symbol": "USDC",
```
[the remark this answers is not in the document]
Comment (manuelwedler) [reply frontend/config/tokens/DAI.json:2]: Ah, I already had the change locally but forgot to modify the commit before pushing

**Review by GabrielBuragev — APPROVED** [review]
Looks good.

To whomever is merging this: Check previous comments and whitelist the "DAI" token accordingly.

**Comment by GabrielBuragev** [comment]
Configured DAI token in the `RequestManager` contracts:
- arbitrum: https://arbiscan.io/tx/0x9ecda9d3b74fd13d0121a1b7b19d51217e6e374638f88ee7fea2974de1d06192
- optimism: https://optimistic.etherscan.io/tx/0xe09585dfeaba8895c02f9ce55247735026b5c90085b04ea47d470bfc48df9974

**Comment by GabrielBuragev** [comment]
Fixed the `mainnet.json` deployment file to reflect the right configuration values for DAI. 
It has 18 decimals so i padded the values accordingly.

**Pull request merged**